src/worldmap.test.ts

```diff
@@ -387,4 +376,46 @@ describe('Worldmap', () => {
     worldMap = new WorldMap(ctrl, document.getElementsByClassName('mapcontainer')[0]);
     worldMap.createMap();
   }
+
+  describe('when apply log is set', () => {
```

Comment:
Would it make sense to write test suites for other `logFunction`s as well?

Currently we are testing only `log1p`

Reply:
Ouch, in fact that field I created to help me choose the best function to differentiate the circles better ... I wasn't going to commit the field altogether hehehe ... but, now that I did, writing tests for it ;)

Reply:
After a math study over the subject, just found out that, the way it is calculated, the base of log doesn't really matters after all. 
Look at the results here: https://www.desmos.com/calculator/1w0qzusjys
So, I AM removing this field.